std/algebra/emulated/sw_bn254/doc.go

```diff
@@ -1,6 +1,6 @@
 // Package sw_bn254 implements G1 and G2 arithmetics and pairing computation over BN254 curve.
 //
-// The implementation follows [[Housni22]]: "Pairings in Rank-1 Constraint Systems".
+// The implementation follows [Housni22]: "Pairings in Rank-1 Constraint Systems".
```

Comment:
Yup, I think `[Housni22]` is correct, then I just wanted to have academic style reference for documentation site (example with `[[Housni22]]`. The inner brackets is for link and outer bracket for display):
<img width="767" alt="image" src="https://user-images.githubusercontent.com/1309050/225970022-72ba43fb-9d45-4fbb-8668-cc05e030d842.png">